### PR TITLE
Kubernetes example and redux

### PR DESCRIPTION
**Pull request by dvaldivia**
scripts to provision a kubernetes cluster locally using `kind` which runs the nodes inside containers.

build the modified

with the script `create-kind.sh`  it will
* create the kind cluster
* configure dashboard
* create folder for the simulated tenant
* install the tenant

### CODE REVIEW

File: k8s/add-volume.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+TENANT_NAME="${1:-tenant-1}"
```
Comment (Alevsk) [inline k8s/add-volume.sh:1]: can you ddd `#!/bin/bash` at the beginning of the file just for convention.

File: k8s/create-kind.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+echo "Provisioning Kind"
```
Comment (Alevsk) [inline k8s/create-kind.sh:1]: can you ddd #!/bin/bash at the beginning of the file just for convention.

File: k8s/deployments/tenant1-env.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tenant-1-env
+  labels:
+    app: tenant-1
+data:
+  MINIO_ACCESS_KEY: "minio"
```
Comment (Alevsk) [inline k8s/deployments/tenant1-env.yaml:8]: For secrets in the future we can use a solution like `envsubst` 
 https://stackoverflow.com/questions/56003777/how-to-pass-environment-variable-in-kubectl-deployment#answer-56009991

**Review by Alevsk — APPROVED** [review]

**Comment by cesnietor** [comment]
are **k8s/deployments/tenant-1.yaml**  and **k8s/deployments/tenant1-env.yaml** planned to be used?

**Comment by dvaldivia** [comment]
@cesnietor yes! there's a dependency between the two, we need the secret first and then the deployment itself

**Pull request merged**